src/components/EntityInfo.vue

```diff
@@ -4,9 +4,13 @@
 
       <!-- Entity Name, Entity Status -->
       <div class="title-container">
-        <div class="mb-1" id="entity-legal-name">
-          <span>{{ entityName || 'Not Available' }}</span>
+        <div v-if="businessId" class="mb-1" id="entity-legal-name" aria-label="Business Legal Name">
```

Comment:
Would it make sense to use <template> here? After you had recommended it to me, i found it made a lot of sense when you want an invisible wrapper to multiple conditional elements. Though, in this case you have a single element and with styling, so it's probably good as it is :) 
Just thinking out loud here!

Reply:
I think there would be little to gain here by using a template element to wrap the v-ifs -- I would still need the divs and spans for styling. But I did fix a few places where there were divs just for the same of a wrapper element.